### ISSUE TITLE
Newest maps disappeared 

### ISSUE THREAD

**Issue by forteller** [umap-project/umap#1045]
Hi. At least my two newest maps have suddenly disappeared. One I created yesterday, and the other one I created some weeks ago, but did edit today.

The URLs where:

https://umap.openstreetmap.fr/no/map/uavhengige-sjapper_859787
https://umap.openstreetmap.fr/no/map/njb-oslo-2023_873112

I put quite a bit of work into these maps, and the njb-oslo one is time critical. So I would appreciate it very much if you could look into this as soon as possible. Thank you so much!

**Comment by claudeaubry** [comment]
Hi. My maps have also disappeared. At least five, created since one month.
Some map names :
- mapName="jusqua-vic-par-le-canal_869901"
- mapName="rejoindre-airbus-de-castanet_865796"
- mapName="boucle-dans-le-razes_857714"

As forteller I spent a lot of work into these maps and there are integrated in sites.

**Comment by Vid-Pogacnik** [comment]
Dear Sirs,
The same story with me. Yesterday uMap was not responding. I log in via OpenStreetMap, which was supposedly not connecting. Some links to my uMaps were not working. A bit later, the links were back, but I noticed that on many maps the last edits were lost.  The most recent work was done on this map:
http://umap.openstreetmap.fr/en/map/kepa_550841#12/46.5167/13.8836
Now it's gone.
Thanks in advance for your help and best regards!
Vid

**Comment by EricZet** [comment]
Same issue here, it is also an open question on: https://help.openstreetmap.org/questions/86726/umap-content-disappeared

**Comment by binndr** [comment]
I have the same issue as of today (19 feb). Only thing that shows up is a map with very old data. 
https://umap.openstreetmap.fr/nl/map/binndr_809360#6/38.823/-0.923

**Comment by Neyliece** [comment]
My map also disappeared yesterday (18 Feb):
https://umap.openstreetmap.fr/fr/map/carte-de-la-france_866511#9/48.7953/3.3217

**Comment by mountainpixel** [comment]
Same issue here, all maps from the last weeks are gone :⁠'⁠(

**Comment by marcoXbresciani** [comment]
Yup, same for my map: https://umap.openstreetmap.fr/it/map/giappone_673259#10/35.4249/139.9658 . It had a hundred places or so, all listed and linked here https://gist.github.com/marcoXbresciani/f4ffad9f2f0cc3857501e06759167b34 but there are no more.

**Comment by grid56126** [comment]
https://umap.openstreetmap.fr/en/user/grid56126/I had at least 8 huge maps
-------- Original message --------From: Marco Bresciani ***@***.***> Date: 19/02/2023  11:24  (GMT+00:00) To: umap-project/umap ***@***.***> Cc: Subscribed ***@***.***> Subject: Re: [umap-project/umap] Newest maps disappeared  (Issue #1045) 
Yup, same for my map: https://umap.openstreetmap.fr/it/map/giappone_673259#10/35.4249/139.9658 . It had a hundred places or so, all listed and linked here https://gist.github.com/marcoXbresciani/f4ffad9f2f0cc3857501e06759167b34 but there are no more.

**Comment by malte1030** [comment]
My latest map also disappeared:
https://umap.openstreetmap.fr/en/map/stadtfuhrungen-woydt_864767
What can I do?
Thanks!

**Comment by grid56126** [comment]
I have lost massive amounts of work, some of mine were 50mb in size, at least 8 maps and seems to have gone back to very early log ins when i first experimented.

Lots of error 502 etc logging in last night, came back all good 2100 G.M.T.

**Comment by Inggareth** [comment]
On Friday I edited map I created about a month ago. Now completely gone. I can see lots of other similar comments. What has happened?

**Comment by ArnaudFc** [comment]
Same problem here, I lost days worth of work on a community map. Everything I added/modified vanished as if I had never been there.
https://umap.openstreetmap.fr/fr/map/carte-de-travail-place-au-velo-angers-alm_394540#15/47.4009/-0.5275

I also lost access to my private map which displays a 404 despite being logged on my account.
https://umap.openstreetmap.fr/fr/map/carte-perso-pav_863786#15/47.4207/-0.5097

**Comment by Xonnext** [comment]
Same problem here. Lost 6 new maps. Hope they can be restored.... Good luck to all (and myself)...

**Comment by grzebo** [comment]
Same problem here, also lost huge amount of work.

**Comment by mburlat68** [comment]
same for me - I lost my last map:
https://umap.openstreetmap.fr/fr/map/randos-hivernales-alpes-et-jura_872522

**Comment by LowMem** [comment]
Same here on map http://umap.openstreetmap.fr/fr/map/eos-flashing-parties_871404

Hope you will be able to recover data.

Keep up the good work!

**Comment by parlconst** [comment]
I've lost over 30 maps created since the end of December 2022.

**Comment by ptaluy** [comment]
Same here, a lot of work is actually lost. Hope that the dev' team will find a solution. 🤞

**Comment by gugalun** [comment]
Same here. I create a map with daily changes. So I knew that all data till December 30 are lost.
Happily I have a backup from February 9.
In future I will make daily backups.

**Comment by grid56126** [comment]
How can you back up offline?



Get BlueMail for Android

**Comment by ptaluy** [comment]
On umap export options, you can export the full map (datas and all umap parameters). With that file saved on your local disk, you can reupload à complete map in a few minutes.

**Comment by DanyaAnderson** [comment]
omg this is a disaster I hope they find them. Now I look like a moron to my team.

I guess I'm downloading at the end of each day.

**Comment by DanyaAnderson** [comment]
> How can you back up offline? Get BlueMail for Android
> […](#)
> On 19 Feb 2023, 17:28, at 17:28, gugalun ***@***.***> wrote: Same here. I create a map with daily changes. So I knew that all data till December 30 are lost. Happily I have a backup from February 9. In future I will make daily backups. -- Reply to this email directly or view it on GitHub: [#1045 (comment)](https://github.com/umap-project/umap/issues/1045#issuecomment-1436046014) You are receiving this because you commented. Message ID: ***@***.***>

Go to Share > Download Data > Full Map Data
I am kicking myself my last download was September

**Comment by DanyaAnderson** [comment]
I am new at this....... do we wait for a (possible) fix or begin rebuilding immediately?

**Comment by grzebo** [comment]
Yes, it would be good to know:
1. Is there still a chance to retrieve the lost maps?
2. Is the problem that caused the disappearance solved or is there a danger of it happening again?

I can start recreating the lost maps, but what if the data will be again rolled back to the current state?

**Comment by deuzeffe** [comment]
https://forum.openstreetmap.fr/t/disparition-de-mes-cartes-sur-umap/12854/7

> Bonjour à tout le monde,
Il y a eu effectivement un souci hier, les supers bénévoles qui gèrent les serveurs sont dessus.
Je n’en sais pas plus pour l’instant, nous vous tiendrons au courant.

> Hello to everyone,
There was indeed a problem yesterday, the great volunteers who manage the servers are on it.
I don't know more for the moment, we'll keep you posted.

https://forum.openstreetmap.fr/t/disparition-de-mes-cartes-sur-umap/12854/10

> Suite des infos, umap est en lecture seule, il n’est plus possible d’enregistrer (attention on ne le sait qu’au moment de sauvegarder…).
Les données pour l’instant sont celles du 27 décembre …
Cela devrait être temporaire, sans savoir quelle durée 

> Further information, umap is read-only, it is no longer possible to 
to save (be careful, you only know it when you save...).
The data for the moment are those of 27 December...
This should be temporary, but we don't know how long it will last

**Comment by gugalun** [comment]
Sorry, if i work with my map i'm online and at the end i make a backup. why shout i make i backup offline?- - -wilhelm wolfHeidelbergwww.wolfx.deAm 19.02.23, 23:15 schrieb grid56126 ***@***.***>:

   How can you back up offline?
   
   
   
   Get BlueMail for Android

**Comment by ArnaudFc** [comment]
https://forum.openstreetmap.fr/t/disparition-de-mes-cartes-sur-umap/12854/22

> Le retour à la normale devrait se faire dans la soirée… si tout se passe comme prévu.

Edit: plutôt dans la nuit que la soirée… 500Gb copiés en 5h, il en reste à peu près autant

> The return to normal should be done in the evening… if all goes as planned.

Edit: rather at night than in the evening… 500Gb copied in 5 hours, there are about as many left

**Comment by EricZet** [comment]
I was just able to view my map. 😃  So I,ve downloaded it so I do have at least one backup of it. 
Hopefully a confirmation is coming soon that everything is back to what it should be.

**Comment by sinanisler** [comment]
nice.
@forteller   you can close it.

**Comment by deuzeffe** [comment]
https://forum.openstreetmap.fr/t/disparition-de-mes-cartes-sur-umap/12854/27

> Fin de restauration et redémarrage d’umap.
En principe tout est ok, rien n’est perdu.
Merci de signaler tout problème (rapidement) !

>End of restoration and restart of umap.
In principle everything is ok, nothing is lost.
Please report any problems (quickly)!

**Comment by forteller** [comment]
Thank you to everyone working hard to fix this!

**Comment by binndr** [comment]
Hi, did it happen again,
I lost my data again

**Comment by ArnaudFc** [comment]
I put a thumbs down on your post because everything seems fine to me (not to be harsh).
However, I did not edit anything since the issue was resolved. So it is possible that recently created data may have been lost.
Has anyone noticed a similar issue as _binndr_?

**Comment by EricZet** [comment]
I’ve made some changes after the issues were resolved. They are still present on my map.

**Comment by Xonnext** [comment]
No, no problems here. Not in the old Maps and not in the newer later designed Maps. It works fine..!Cees GeuzeOp 24 feb. 2023 om 12:19 heeft ArnaudFc ***@***.***> het volgende geschreven:﻿
I put a thumbs down on your post because everything seems fine to me (not to be harsh).
However, I did not edit anything since the issue was resolved. So it is possible that recently created data may have been lost.
Has anyone noticed a similar issue as binndr?
